package ExeCeption;

public class Arith extends Exception{

	public static void main(String[] args) {
		// TODO Auto-generated method stub

	}

	public Arith() {
		super();
		// TODO Auto-generated constructor stub
	}

	public Arith(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace) {
		super(message, cause, enableSuppression, writableStackTrace);
		// TODO Auto-generated constructor stub
	}

	public Arith(String message, Throwable cause) {
		super(message, cause);
		// TODO Auto-generated constructor stub
	}

	public Arith(String message) {
		super(message);
		// TODO Auto-generated constructor stub
	}

	public Arith(Throwable cause) {
		super(cause);
		// TODO Auto-generated constructor stub
	}
	

}
